Kwenta/kwenta
# Enabled the BNB, DOGE and Debt Ratio market

## Description
Add three new markets: DOGE-PERP, BNB-PERP, and DEBT-PERP.

Note: The decimal of DEBT-PERP is different between subgraph (`latestRates` and `candles`) and contract (`FuturesMarketData.allMarketSummaries`). We have to patch the output from the subgraph as a workaround.

## Related issue
#1309
#1326 

## Motivation and Context


## How Has This Been Tested?
1. Open the market page and switch to DOGE, BNB and DebtRatio market
2. Open the dashboard page and see the DOGE, BNB and DebtRatio on the list

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4819006/187092474-85e65a7e-cb10-4dde-a3c8-3aaa9a660ffd.png)
![image](https://user-images.githubusercontent.com/4819006/187092494-738cd382-7f01-4a75-a316-0bf513d2b726.png)
![image](https://user-images.githubusercontent.com/4819006/187092786-bbbc6e96-a2c3-4523-b2fd-eca47ddb36bf.png)
